src/org/usfirst/frc4904/robot/humaninterface/drivers/AlignAssist.java

```diff
@@ -15,8 +15,8 @@
 	public AlignAssist(GearAlign gearAlign, EnableableModifier fineModifier) {
 		this.gearAlign = gearAlign;
 		this.fineModifier = fineModifier;
-		enableFineModifier = new SetEnableableModifier(fineModifier, true);
-		disableFineModifier = new SetEnableableModifier(fineModifier, false);
+		enableFineModifier = new SetEnableableModifier(true, fineModifier);
```

Comment:
Is this change correct? The construction of SetEnableableModifier takes in the modifier and then a boolean. Plz explain.